src/plugins/vis_augmenter/public/plugin.ts

```diff
@@ -33,14 +47,27 @@ export class VisAugmenterPlugin
 
   public setup(
     core: CoreSetup<VisAugmenterStartDeps, VisAugmenterStart>,
-    { data, expressions }: VisAugmenterSetupDeps
+    { expressions }: VisAugmenterSetupDeps
   ): VisAugmenterSetup {
     expressions.registerType(visLayers);
+    setUISettings(core.uiSettings);
     return {};
   }
 
-  public start(core: CoreStart, { data }: VisAugmenterStartDeps): VisAugmenterStart {
+  public start(
+    core: CoreStart,
+    { uiActions, embeddable, data, visualizations }: VisAugmenterStartDeps
+  ): VisAugmenterStart {
     setUISettings(core.uiSettings);
```

Comment:
Let me get back to you on that. This was from conflict resolutions too. We are finalizing where we need these settings, since there's some tricky dependency situations to handle when consumed by plugins, but using plugin functionality in other plugins (like Visualizations) that aren't dependent on the plugins.

Reply:
We can remove from setup(). The dependency issue is fixed for now in #4139

Reply:
removed